Felfel.Logging/LogEntryDto.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Felfel.Logging
+{
+    /// <summary>
+    /// Serializable representation of a <see cref="LogEntry"/>.
+    /// </summary>
+    public class LogEntryDto
+    {
+        internal const string DataPropertyPlaceholderName = "@logentry_data";
+
+        [JsonProperty("@timestamp")]
+        public DateTimeOffset Timestamp { get; set; }
+
+        /// <summary>
+        /// The context / application area where the logged
+        /// message originates from. Used to simplify filtering
+        /// and identification of issues in code.
+        /// </summary>
+        public string Context { get; set; }
```

Comment:
do we care about the casing of the keys for the logs?

Reply:
We do. The JSON serializer is set up to convert everything to `snake_case`.

Reply:
missed that. sorry.